src/internalBinding/timers.cc

```diff
@@ -36,21 +36,14 @@ static bool enqueueWithDelay(JSContext *cx, unsigned argc, JS::Value *vp) {
   return true;
 }
 
-// TODO (Tom Tang): move argument checks to the JavaScript side
```

Comment:
I think on line 35 above we should return an Int32. This would also be closer to the specs: https://developer.mozilla.org/en-US/docs/Web/API/setTimeout#return_value